.github/workflows/release-linux.yml

```diff
@@ -23,30 +23,57 @@ jobs:
       run: |
         tar -xf source.tar.gz --strip-components=1
 
+    - name: Enable vcpkg cache
+      uses: actions/cache@v3
+      with:
+        path: /vcpkg/installed
+        key: ubuntu-20.04-vcpkg-release-0 # Increase the number whenever dependencies are modified
+        restore-keys: |
+          ubuntu-20.04-vcpkg-release
+
     - name: Install dependencies
       run: |
-        echo "::group::Install dependencies"
+        echo "::group::Install system dependencies"
         yum install -y \
-          fontconfig-devel \
-          freetype-devel \
-          libcurl-devel \
           libicu-devel \
-          libpng-devel \
-          lzo-devel \
-          SDL2-devel \
+          perl-IPC-Cmd \
```

Comment:
...perl?

Reply:
Yup. Really. It is needed to install one of the vcpkg dependencies. Don't ask .. just .. don't .. ask ...

Reply:
Nice. Maybe a comment?

Reply:
Ugh. Okay. Kept it generic, but you can do the math :P